tests/integration/api/test_sc_reports.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+import xml.etree.ElementTree as ET
+
+from tests.base import BaseTest
+from tenable_io.api.models import ScReport
+
+
+class TestScReportsApi(BaseTest):
+
+    @pytest.fixture(scope='class')
+    def container_report(self, client):
+        containers = client.sc_containers_api.list()
+        assert len(containers) > 0, u'Container list has at least one container for testing.'
```

Comment:
As we talked about earlier, we cannot make assumption about pre-conditions. Unfortunately that restricts the type of tests we can write.